.github/workflows/test-and-deploy.yml

```diff
@@ -29,18 +30,28 @@ jobs:
       - name: Run Unit Tests
         run: make test-docker version=${{ matrix.php }}
 
+    deploy:
+      name: Deploy
+      if: success() && github.ref_type == 'tag'
+      needs: [ test ]
+      steps:
+        - name: Create GitHub Release
```

Comment:
Also need to include the `sendgrid-php.zip` asset. It's generated with `make bundle`: https://github.com/sendgrid/sendgrid-php/blob/1daa621985dd49da0471485f96dab4ab0e6338c6/Makefile#L41

Reply:
Added the steps to install and and build the library artifact